apps/site/lib/site_web/views/trip_plan_view.ex

```diff
@@ -645,21 +652,16 @@ defmodule SiteWeb.TripPlanView do
     )
   end
 
-  @spec get_highest_one_way_fare(TripPlan.Itinerary.t()) :: integer
-  def get_highest_one_way_fare(itinerary) do
+  @spec get_one_way_total_by_type(TripPlan.Itinerary.t(), atom) :: integer
```

Comment:
The `atom` type is fine, but if you wanted you could also define a new type to describe the options since they are so limited. Maybe something like:

    @type fare_type :: :highest_one_way_fare | :lowest_one_way_fare | :reduced_one_way_fare

Reply:
:thumbsup: Yes, makes sense, I didn't think of that.